modules/seqexec/engine/src/test/scala/seqexec/engine/packageSpec.scala

```diff
@@ -178,15 +177,17 @@ class packageSpec extends FlatSpec with NonImplicitAssertions {
     )
     val p = Stream.eval(IO.pure(Event.start(seqId, user, UUID.randomUUID())))
 
-    executionEngine.process(p)(s0).take(1).compile.last.unsafeRunSync.map(_._2)
+    //take(3): Start, Executing, Paused
+    executionEngine.process(p)(s0).take(3).compile.last.unsafeRunSync.map(_._2)
```

Comment:
Previous to the merge, this would finish naturally after processing the single value in the input Stream and the other Streams generated internally. Because now there is a queue inside, it never terminates, so it is necessary to predict how many events will take to reach the result.